solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LTRScoringModel.java

```diff
@@ -108,7 +108,7 @@ public static LTRScoringModel getInstance(SolrResourceLoader solrResourceLoader,
         SolrPluginUtils.invokeSetters(model, params.entrySet());
       }
     } catch (final Exception e) {
-      throw new ModelException("Model type does not exist " + className, e);
+      throw new ModelException("Model creation failed for " + className, e);
```

Comment:
minor: how about not saying "creation" but perhaps "loading" or something to that effect? technically the "creation" i.e. storage of the model's JSON succeeded (if i remember things right) but it's the "loading" or "using" of the model's JSON that encounters the error. so the user seeing the error needs to look at the content of what they uploaded but not at the mechanics of how the uploaded/created the model.

Reply:
agreed!